0002_child_test.js

```diff
@@ -559,12 +584,57 @@ describe("Test add-new forms:", () => {
          .should("exist")
          .click();
 
-      cy.get(".webix_warn").find(".webix_button").should("be.visible").click({ multiple: true, force: true });
+      cy.get(".webix_warn")
+         .find(".webix_button")
+         .should("be.visible")
+         .click({ multiple: true, force: true });
       // should contain new data in grid
       cy.get(cyInterfaceCHILD.page.basicInfo.page.files.grid).contains(text);
       checkForm("basicInfo", "files", "label");
    });
 
+   // Checking the label of Initial Asset log
+   it("Test the label 'No Item Reported' of Initial Asset log", () => {
+      // Cannot click on already viewed child page
+
+      // cy.get(cyInterfaceCHILD.tab.basicInfo).should("be.visible").click();
+
+      cy.get(cyInterfaceCHILD.page.basicInfo.tab.admitInfo)
+         .should("exist")
+         .click();
+      cy.get('[data-cy="detail connected Initial Assets 6daf2109-4bbe-4003-b69d-bed120b83ec2 8db15d0b-fd78-4b2b-975a-304b68a43054"]')
+         .should("exist");
+      cy.get('[data-cy="menu-item Admission Info_3a46 cf368115-3a46-434b-b44e-6104b9bdb592 f076f340-9d27-45cf-b75e-c0930506235e"]')
+         .should("exist")
+         .click()
+         .wait(8000);  // Wait for the details showing
```

Comment:
Should we use `cy.get(...).should("exist")` instead?

Reply:
Already Updated